django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -195,4 +195,12 @@ Requires core.js, SelectBox.js and addevent.js.
         }
     };
 
+    addEvent(window, "load", function(e) {
```

Comment:
I think to match the other code I'll update it if you think the jquery solution is preferred